service/k8sclient/k8sclient.go

```diff
@@ -704,6 +734,16 @@ func (c *K8sClient) getPerconaXtraDBClusters(ctx context.Context) ([]XtraDBClust
 				{size: cluster.Status.PXC.Size, ready: cluster.Status.PXC.Ready},
 			},
 		}
+		if cluster.Spec.ProxySQL != nil {
```

Comment:
we can get rid of the else branch here too

```go

if cluster.Spec.ProxySQL != nil {
	val.ProxySQL = &ProxySQL{
		DiskSize:         c.getDiskSize(cluster.Spec.ProxySQL.VolumeSpec),
		ComputeResources: c.getComputeResources(cluster.Spec.ProxySQL.Resources),
	}
    res[i] = val
    return res, nil
}

val.HAProxy = &HAProxy{
	ComputeResources: c.getComputeResources(cluster.Spec.HAProxy.Resources),
}
res[i] = val
return res, nil
````

Reply:
I used `continue` to get rid of the else branch.